frontend/src/hooks/common/useCitySuggestion.ts

```diff
@@ -1,21 +1,19 @@
+import { useQueryClient } from '@tanstack/react-query';
 import type { CityData } from '@type/city';
 import { useEffect, useState } from 'react';
 
 import { makeRegexByCho } from '@utils/cityFilter';
 
-import { useCityQuery } from '@hooks/api/useCityQuery';
-
 export const useCitySuggestion = ({ onItemSelect }: { onItemSelect: (item: CityData) => void }) => {
-  const { citiesData } = useCityQuery();
-  console.log(citiesData);
+  const cityData = useQueryClient().getQueryData<CityData[]>(['city']);
```

Comment:
바로 `useQueryClient().getQueryData...` 처럼 적는 것 보다 아래처럼 적는 것이 더 깔끔한 것 같아요


```suggestion
const queryClient = useQueryClient();
const cityData = queryClient.getQueryData<CityData[]>(['city']);
```